Makefile

```diff
@@ -173,7 +173,6 @@ analyze: $(JS)
 ifeq ($(DEV), ashell)
 	@cat fragments/prj.mdef.dev >> prj.mdef
```

Comment:
shouldn't prj.mdef.dev and prj.mdef.heap  be removed as well? or move to prj.conf equivalent?